addons/core/translations/en-us.yaml

```diff
@@ -86,3 +86,7 @@ onboarding:
       learn: Learn about Boundary Desktop app
       cli: Learn about Boundary command-line (CLI)
       api: Learn about Boundary API
+proxy-url:
+  options:
+    address-port: Address & port
+    ssh: SSH
```

Comment:
I added these translations here as they are not directly tied to sessions. Resources didn't feel like the best place but happy to take suggestions.

Reply:
This text is specific to the component and more so has to do with the format of session proxy info, not the session itself. Maybe that is an impractical approach though.

Reply:
sounds resource-specific to me